test/functional/trends_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class TrendsControllerTest < ActionController::TestCase
+  setup do
+    User.current = users(:admin)
```

Comment:
Avoid doing this as it leaks into new tests and makes things unpredictable.  I don't think it's needed for this test so can be removed, but if it were, you could use as_user/as_admin as necessary.